common/src/main/java/com/wynntils/utils/MathUtils.java

```diff
@@ -68,11 +68,10 @@ public static int integerFromRoman(String numeral) {
     }
 
     public static float map(float sourceNumber, float fromA, float fromB, float toA, float toB) {
-        float deltaA = fromB - fromA;
-        float deltaB = toB - toA;
-        float scale = deltaB / deltaA;
-        float negA = -1 * fromA;
-        float offset = (negA * scale) + toA;
-        return (sourceNumber * scale) + offset;
+        return MathUtils.lerp(toA, toB, MathUtils.inverseLerp(fromA, fromB, sourceNumber));
+    }
+
+    public static float mag(float x, float y) {
```

Comment:
What is `mag`? This is a normal euclidean distance?

Reply:
https://en.wikipedia.org/wiki/Magnitude_(mathematics)

Reply:
Aha, mag as in magnitude. I was so sure it meant me. ;-)

Joking aside, I think `distance` is a much better name. Using "magnitude" in this sense is for me strictly reserved to complex numbers, and then the arguments would not have been named "x" and "y" (and we do not do any complex arithmetic).

Also, do we not already *have* a `distance` function?

Reply:
Magnitude is also used in the case of vectors.

Reply:
Though I think there is already a method